packages/server/api/test/unit/ai/prompts.service.test.ts

```diff
@@ -0,0 +1,183 @@
+const mockFetch = jest.fn();
+global.fetch = mockFetch;
+
+const readFileMock = jest.fn();
+jest.mock('fs/promises', () => ({
+  readFile: readFileMock,
+}));
+
+const getMock = jest.fn();
+jest.mock('@openops/server-shared', () => ({
+  logger: {
+    error: jest.fn(),
+  },
+  system: {
+    get: getMock,
+  },
+  AppSystemProp: {
+    INTERNAL_PROMPTS_LOCATION: 'INTERNAL_PROMPTS_LOCATION',
+  },
+}));
+
+import { getSystemPrompt } from '../../../src/app/ai/chat/prompts.service';
+
+describe('getSystemPrompt', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should load aws cli block prompt from cloud', async () => {
```

Comment:
these tests are all identical except for "cloud" / "local", maybe add a common function?